apps/arweave/src/ar_webhook.erl

```diff
@@ -29,10 +20,27 @@
 	{<<"content-type">>, <<"application/json">>}
 ]).
 
+-define(MAX_TX_OFFSET_CACHE_SIZE, 100000).
+
+-record(tx_offset_cache, {
+	txid_timestamp = #{},
```

Comment:
```suggestion
	txid_timestamp = #{}, %% map TXID to timestamp
```

Reply:
what do you think about adding `to` in the map names to clarify which element is the key vs. value? More important for `txid_start_offset_end_offset` than `txid_timestamp`